pypi_formula_mappings.json

```diff
@@ -415,6 +415,9 @@
   "liquidctl": {
     "exclude_packages": ["pillow"]
   },
+  "literate-git": {
+    "package_name": "literategit"
```

Comment:
`update-python-resources` should be able to infer this from the url

Reply:
oh, ok, that is good to know 👍